### PR TITLE
copr: support push bit column down

### PR DESCRIPTION
**Pull request by Yisaer**
### What is changed and how it works?

Issue Number: ref https://github.com/pingcap/tidb/issues/30738

What's Changed:

Support push bit column down in tikv

```commit-message
```

### Check List 

Tests 

- Unit test
- Manual test (add detailed scripts or steps below)

[test document](https://docs.google.com/document/d/1aiXYERBaQ1aNIZpsUh3tVE8jIYfXT-5954JOsYJAR5U/edit?usp=sharing) in Chinese

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/tidb_query_expr/src/impl_cast.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+use byteorder::{BigEndian, ByteOrder};
```
Comment (zhongzc) [inline components/tidb_query_expr/src/impl_cast.rs:3]: Better to move this line to the third imports block.

**Review by zhongzc — APPROVED** [review]

**Review by longfangsong — APPROVED** [review]

**Pull request merged**